content/blog/recipes-1-3-0/index.Rmd

```diff
@@ -0,0 +1,183 @@
+---
+output: hugodown::hugo_document
+
+slug: recipes-1-3-0
+title: recipes 1.3.0
+date: 2025-04-22
+author: Emil Hvitfeldt
+description: >
+    This release brings multiple exciting features and streamlines many recipe steps.
```

Comment:
This feels a tad clickbait-y 😅

Reply:
added more information here https://github.com/tidyverse/tidyverse.org/pull/730/commits/be40d9d626ed8618b3ca9e47bed86274234befbd